### ISSUE TITLE
instance segmentation is all 0's

### ISSUE THREAD

**Issue by bradyz** [philkr/pystk#26]
in the master branch, `race.render_data[0].instance` always gives all 0's

to reproduce, edit
https://github.com/philkr/pystk/blob/master/examples/hockey_gamestate.py#L50

to

```
    uis = [gui.UI([gui.VT['SEMANTIC']])]
```

then run

```
python3 -m examples.hockey_gamestate
```

**Comment by qw3rtman** [comment]
Some findings for `render_data[0].instance`:

c30fc38c8 — all 0’s
84b13e2cc — all 0’s
db13e8089 — all 0’s
fcd03a93b — choppy segmentation (see below)
![image](https://user-images.githubusercontent.com/1139621/74598235-8d283c80-5033-11ea-84b4-92f71e407592.png)

98c4daeb4 — working as expected

**Comment by philkr** [comment]
fixed in 235eb0069385b6d06212eaedbfd09d26cf5e127d . Please reopen of still broken.
